test/acceptance/features/editDogDetailsvalidations.feature

```diff
@@ -23,6 +23,13 @@ Feature: Edit dog details
         Then I expect that element "form" contains the text "Microchip numbers must be 15 numbers long"
         Then I expect that element "form" contains the text "Microchip numbers can only contain numbers"
 
+    Scenario: Enter microchip number already exists in the database for another dog
+        When I set "345677654366666" to the inputfield "#microchipNumber"
+        And I set "345677654355555" to the inputfield "#microchipNumber2"
+        And I click on the element "button=Save details"
+        Then I expect that element "form" contains the text "The microchip number already exists"
+        And I expect that element "form" contains the text "The microchip number already exists"    
```

Comment:
Did you mean to duplicate this line (almost) ?

Reply:
Removed the duplicate line and added separate tests to check each microchip number field separately.